# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../nepenthes/libprelude.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- nepenthes-0.1.7/modules/log-prelude/log-prelude.hpp.orig	2006-10-16 02:45:08.000000000 +0200
+++ nepenthes-0.1.7/modules/log-prelude/log-prelude.hpp	2006-10-16 02:45:25.000000000 +0200
@@ -30,7 +30,7 @@
 #include "config.h"
 
 #ifdef HAVE_LIBPRELUDE
-#include <prelude.h>
+#include <libprelude/prelude.h>
 #endif
 
 #include <string>
